apps/meteor/app/api/server/v1/cloud.ts

```diff
@@ -122,3 +123,23 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'cloud.syncWorkspace',
+	{
+		authRequired: true,
+		permissionsRequired: ['view-privileged-setting'],
```

Comment:
we already have a Meteor method for calling `syncWorkspace` which currently required `manage-cloud` permission:

https://github.com/RocketChat/Rocket.Chat/blob/2741b04f17d469bea9a66460a5c7b72c0844a713/apps/meteor/app/cloud/server/methods.ts#L105

I think this should required the same permission

Reply:
You are right! Thanks